### PR TITLE
update logs of notification module.

### PR DESCRIPTION
**Pull request by ahmetoz**
* details: https://docs.adyen.com/development-resources/webhooks/understand-notifications

### CODE REVIEW

**Review by JudeNiroshan — APPROVED** [review]

**Comment by JudeNiroshan** [comment]
```
 {
  err: {
   message: "undefined is not iterable (cannot read property Symbol(Symbol.iterator))"    
   name: "TypeError"    
   stack: "TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
    at /workspace/node_modules/p-map/index.js:23:28
    at new Promise (<anonymous>)
    at module.exports (/workspace/node_modules/p-map/index.js:12:9)
    at Object.processNotifications (/workspace/src/handler/notification/notification.handler.js:12:9)
    at exports.notificationTrigger (/workspace/index.js:21:19)
    at /layers/google.nodejs.functions-framework/functions-framework/node_modules/@google-cloud/functions-framework/build/src/invoker.js:100:17
    at processTicksAndRejections (internal/process/task_queues.js:79:11)"    
  }
  hostname: "localhost"   
  level: 50   
  msg: "Unexpected error when processing event"   
  name: "ctp-adyen-integration-notifications"   
  notification: [
   0: {
    NotificationRequestItem: {
     additionalData: {
      hmacSignature: "cjiTz03EI0jkkysGDdPJQdLbecRVVU/5jm12/DTFEHo="       
     }
     amount: {
      currency: "EUR"       
      value: 10100       
     }
     eventCode: "AUTHORISATION"      
     eventDate: "2019-01-30T18:16:22+01:00"      
     merchantAccountCode: "CommercetoolsGmbHDE775"      
     merchantReference: "8313842560770001"      
     operations: [
      0: "CANCEL"       
      1: "CAPTURE"       
      2: "REFUND"       
     ]
     paymentMethod: "visa"      
     pspReference: "test_AUTHORISATION_1"      
     success: "true"      
    }
   }
  ]
  pid: 1   
  v: 0   
 }
```

File: notification/src/utils/commons.js [hunk 1]
```diff
@@ -18,7 +18,22 @@ function sendResponse(response, statusCode = 200, headers, data) {
...
 
+function getNotificationForTracking(notification) {
+  if (notification && notification.NotificationRequestItem) {
+    const notificationRequestItem = notification.NotificationRequestItem
+    return {
+      eventCode: notificationRequestItem.eventCode,
+      eventDate: notificationRequestItem.eventDate,
+      pspReference: notificationRequestItem.pspReference,
+      success: notificationRequestItem.success,
+      // reason: notificationRequestItem.reason // not sure about that.
```
Comment (JudeNiroshan) [inline notification/src/utils/commons.js:29]: Can we uncomment this?

Comment (salander85) [reply notification/src/utils/commons.js:29]: It can be not existing... do you think it makes sense to put it?

**Review by JudeNiroshan — APPROVED** [review]
LGTM

File: notification/src/utils/commons.js (hunk 1 above)
Comment (salander85) [reply notification/src/utils/commons.js]: dead3ff

**Comment by ahmetoz** [comment]
Merged into the #528

**Pull request closed without merging**